backend/core/src/seeder/seed.ts

```diff
@@ -381,6 +384,10 @@ async function seed() {
       ])
     }
   }
+  // process duplicate check
+  console.log("processing duplicates check")
```

Comment:
🧹

Reply:
I think it may be helpful to keep these in for looking at the logs, at least initially.